Fabric.Authorization.API/scripts/AzureAD-Groups-Migration.ps1

```diff
@@ -26,7 +26,11 @@ Import-Module ActiveDirectory
 $fabricInstallUtilities = "$PSScriptRoot\Fabric-Install-Utilities.psm1"
 if (!(Test-Path $fabricInstallUtilities -PathType Leaf)) {
     Write-DosMessage -Level "Warning" -Message "Could not find fabric install utilities. Manually downloading and installing"
-    Invoke-WebRequest -Uri https://raw.githubusercontent.com/HealthCatalyst/InstallScripts/master/common/Fabric-Install-Utilities.psm1 -Headers @{"Cache-Control" = "no-cache"} -OutFile $fabricInstallUtilities
+    $originalProgressPreference = $progressPreference
```

Comment:
Could this be made into a more generic function instead and called in one line vs having 5 everywhere we call this?

Reply:
I did consider that but discovered the parameters passed to Invoke-WebRequest vary between call occurrence. I'll take another look and see if there's a clean way to accomplish this. At a minimum I can inventory the parameters used.

Reply:
Main variance is around the setting of the "Cache-Control" header (12 out of 22 occurrences). I can create a wrapper function... next concern is where should this live (e.g. Install-Authorization-Utilities.psm1)?